core/src/main/java/org/apache/calcite/materialize/Lattice.java

```diff
@@ -501,6 +514,11 @@ public boolean isAlwaysMeasure(Column column) {
     return !columnUses.get(column.ordinal).contains(false);
   }
 
+  /** Specifies the dialect used to generate SQL query when populate tile. */
+  public static SqlDialect dialectUsedToGenerateSqlWhenPopulateTile() {
```

Comment:
this naming is really just in-consistent with calcite. 
what makes it worse is that this is a public static method. 

is getDialect significantly worse ?